detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentManager.kt

```diff
@@ -89,7 +89,7 @@ class TorrentManager private constructor(
                     delay(100)
                 }
                 profile.updateEntryDuration(
-                    createKey(content.handle.makeMagnetUri(), content.fileIndex),
+                    createKey(MagnetLink.hashFromMagnet(content.handle.makeMagnetUri()), content.fileIndex),
```

Comment:
I think you can just get it from the content object, since it is a TorrentHandler I think